datalad_registry/templates/overview.html

```diff
@@ -35,11 +35,12 @@
 </head>
 <body>
 <div id="nav-logo">
-  <a href="http://registry.datalad.org/"><img src="/static/pics/registry_logo.svg" alt="Registry Logo"/></a>
+  <a href="{{ url_for('.overview') }}"><img src="/static/pics/registry_logo.svg" alt="Registry Logo"/></a>
 </div>
 <div id="datalad-registry">
   <div class="content">
-    <form action="{{ url_for('.overview') }}" formmethod="get">
+    <form action="{{ url_for('.overview') }}" method="get">
+      <label for='search' class="sr-only">Search</label>
```

Comment:
Do we really need it there, i.e.does it convey some additional information which otherwise not there in subsequent "Search query" and "Search" button?

Reply:
ok, let's hope it would come handy ;-)